modules/Components/src/client/ACASFormChemicalStructure.coffee

```diff
@@ -64,9 +64,9 @@ class ACASFormChemicalStructureExampleController extends Backbone.View
 		mol = @sketcher.getChemDoodleJSON()
 		alert mol
 
-class ACASFormChemicalStructureController extends Backbone.View
+class MaestroChemicalStructureController extends Backbone.View
```

Comment:
I'm curious why this was renamed

Reply:
9 days ago you commented "It's confusing to me that the Maestro sketcher is named "ACASForm". Can we just rename it MaestroChemicalStructureController?" 

Were you referring to something else?

Reply:
Ah, okay. Sorry for the confusion here. I want the class names to reflect the actual sketcher.
It looks like we have "ACASForm", "Ketcher", and "MarvinJS" defined here. If I look at the "searchFrameURL" for the "ACASForm" sketcher, I find this: https://github.com/mcneilco/acas/blob/release/2022.1.x/modules/Components/src/client/ACASFormChemicalRegStructure.jade which shows that it is actually using the "ChemDoodleWeb" sketcher.

So I think we should rename this one to reflect that it uses "ChemDoodleWeb". As far as I can tell, this ACASFormChemicalStructure.coffee has no implementations that use the Maestro sketcher. But we can add that in a future PR when we do our focused work on sketcher support.